packages/widgetbook/lib/src/addons/text_scale_addon/text_scale_addon.dart

```diff
@@ -19,35 +20,26 @@ class TextScaleAddon extends WidgetbookAddOn {
               _wrapperBuilder(context, child, routerData, setting),
           builder: _builder,
           providerBuilder: _providerBuilder,
-          selectionCount: _selectionCount,
           getQueryParameter: _getQueryParameter,
         );
 }
 
-String _getQueryParameter(BuildContext context) {
-  final selectedItems =
-      context.read<TextScaleSettingProvider>().value.activeTextScales;
+Map _getQueryParameter(BuildContext context) {
```

Comment:
This should be a `Map<String, dynamic>` if you just do `Map`, it'll be a `Map<dynamic, dynamic>` which is not what we want.

Reply:
Its best to change this in the `WidgetbookAddOn` class